packit/distgit.py

```diff
@@ -88,6 +90,22 @@ def __repr__(self):
             f"absolute_specfile_path='{self.absolute_specfile_path}')"
         )
 
+    @classmethod
+    def clone(
+        cls,
+        config: Config,
+        package_config: CommonPackageConfig,
+        path: Path,
+        branch,
+    ):
+        # TODO: use fedpkg for this, or even better, the lp property below
+        clone_fedora_package(
+            package_config.downstream_package_name, path, branch=branch
+        )
+        lp = LocalProject(working_dir=path)
+        i = cls(config, package_config, local_project=lp)
+        return i
```

Comment:
Can we have a better name or return that instance directly?

Reply:
yup